docs/data-management/definitions/fact-sql.md

```diff
@@ -1,55 +1,46 @@
 # Fact SQL
 
-Fact SQL queries are queries that correspond to events.
+Fact SQL queries define the metric events to analyze in Eppo. Each Fact SQL should return the following columns:
 
-When you write Fact SQL's, you're pulling data from the data warehouse that correspond to specific events that serve as input to metrics. For example, sign-ups, activations, net subscriptions, etc.
+1. **Entity ID(s)** - a set of entity-level identifiers used to join to assignment logs. Examples include `user_id`, `company_id`, or `anonymous_id`.
+2. **Timestamp** - the timestamp that the event occurred.
+3. **Fact Values (optional)** - Optional values associated with the event (revenue amount, minutes watched, etc.).
+4. **Fact Properties (optional)** - Optional properties associated with the event (`purchase_type`, etc.). Note that properties that are 1:1 with subjects should be defined in the [Assignment](/data-management/definitions/assignment-sql) or [Entity Property](/data-management/definitions/property-sql) SQL definition.
 
-## Creating a Fact SQL
-
-1. Navigate to **Definitions** and click **Create Definition SQL**
-
-![Create Definition SQL](/img/building-experiments/create-definition-sql.png)
-
-2. Click **Fact SQL**
-
-![Create Fact SQL](/img/building-experiments/create-fact-sql.png)
 
-3. Select **User**
 
-**User** is the default entity in Eppo but you can also create your own custom entity and select that here.
-
-4. Name your Fact SQL
-
-![Name Fact SQL](/img/building-experiments/name-fact-sql.png)
-
-5. Write SQL in the SQL editor to pull events data from the data warehouse
+## Creating a Fact SQL
 
-Recall that that you should have (potentially multiple) event tables in your data warehouse with certain column types.
+1. Navigate to **Definitions**, click **Create Definition SQL**, and select **Create Fact SQL**
 
-In this step, you're going to write SQL to pull that data.
+![Create SQL Definition](/img/building-experiments/create-definition-sql.png)
 
-![Name Fact SQL](/img/building-experiments/add-fact-sql-query.png)
+2. Enter a name for the Fact SQL
+3. Write SQL that returns the fact source and hit **Run**
+4. Map entity IDs and timestamp columns
 
-6. Annotate the columns that you've selected from the data warehouse
+![Create Fact SQL](/img/building-experiments/create-fact-sql.png)
 
-![Annotate Fact SQL](/img/building-experiments/annotate-fact-sql-columns.png)
+## Adding Facts
 
-In the example above, Eppo has already automatically determined that the `timestamp of creation` column is `TS` and the relevant `entity id` column is `USER_ID`.
+Once entity IDs and timestamps have been added, click "Add Fact" to map fact value columns. If each record should be treated as one event, select `Each Record` instead of a column name.
```

Comment:
```suggestion
Once entity IDs and timestamps have been added, click "Add Fact" to map fact value columns. If each row should be treated as one event, select `Each Record` instead of a column name.
```